Sources/SPTDataLoaderSwift/DataLoader.swift

```diff
@@ -23,59 +23,23 @@ import Foundation
 
 /// A protocol that provides data request handling.
 public protocol DataLoader {
-    /// Performs a request and provides a `SPTDataLoaderResponse`.
-    /// - Parameter request: The `SPTDataLoaderRequest` to perform.
-    /// - Parameter completionHandler: The callback closure invoked upon response.
-    /// - Returns: A token capable of cancelling the request, or `nil` if the request is invalid.
-    @discardableResult
-    func request(
-        _ request: SPTDataLoaderRequest,
-        completionHandler: @escaping (SPTDataLoaderResponse) -> Void
-    ) -> SPTDataLoaderCancellationToken?
-
-    /// Performs a request and provides a `Response` containing the optional response data.
-    /// - Parameter request: The `SPTDataLoaderRequest` to perform.
-    /// - Parameter completionHandler: The callback closure invoked upon response.
-    /// - Returns: A token capable of cancelling the request, or `nil` if the request is invalid.
-    @discardableResult
-    func request(
-        _ request: SPTDataLoaderRequest,
-        completionHandler: @escaping (Response<Data?, Error>) -> Void
-    ) -> SPTDataLoaderCancellationToken?
+    /// Creates a request that can be used to retrieve the contents of a URL.
+    /// - Parameter url: The `URL` for the request.
+    /// - Parameter sourceIdentifier: The identifier for the request source. May be `nil`.
+    func request(_ url: URL, sourceIdentifier: String?) -> Request
```

Comment:
Is this a factory method for a `Request` object or does it perform a request? The comment block doesn't make this clear.

Reply:
Good point. The comment should specify that it creates a `Request` not a request. Will update.